Utilities/Thread.cpp

```diff
@@ -2709,9 +2709,18 @@ u64 thread_ctrl::get_affinity_mask(thread_class group)
 						break;
 					case 16:
 						// 1700, 1800, 2700, TR 1900X family
-						ppu_mask = 0b1111111100000000;
-						spu_mask = ppu_mask;
-						rsx_mask = 0b0000000000111100;
+						if (g_cfg.core.thread_scheduler == thread_scheduler_mode::ht_off && g_cfg.core.thread_scheduler != thread_scheduler_mode::os)
```

Comment:
nitpick. you can simplify all of these checks in this file to check for old scheduler in the if and then put your code in the else block

Reply:
ah. I didn't see the different == != . never mind then

Reply:
ah.... actually.
the second check is completely obsolete in all occasions

Reply:
I was wondering about that, yes. I added it there because I was unsure.

Reply:
if the thread_scheduler equals ht_off, then it cannot be any other scheduler, so the other check is always true.

Reply:
And if it hits the else statement? Else should only be hit with the old scheduler.

Reply:
Though I guess this code shouldn't be called at all with the os scheduler if I'm reading it right.

Reply:
well. the question is, what do you wanna achieve